src/components/WalletBackupForm/WalletBackupForm.js

```diff
@@ -113,7 +113,7 @@ export function WalletBackupForm({
           type='button'
           theme='general'
           className={walletBackupFormStyle.button}
-          onClick={handleDownload}
+          onClick={handleDownload(values.data, passphrase, derivationPath)}
```

Comment:
Bad practice, you create new function on every props check resulting in rerender.

You need to memoize the resulting function or make it method of class instance.

Reply:
we always have just one set of these props

Reply:
It does not matter. You still create a new function. And react always calls render (or function component body in your case) and then makes strict comparison of props by reference – and you will always have a new reference here.

Try in browser console, for example:
```js
function foo(i, j) {
  return () => i + j
}

console.log(foo(0, 1) === foo(0, 1))
```

Arguments are the same, but results are not equal.

Reply:
Sure, I know it.
But especially for this component it absolutely is not a problem.

Reply:
True.